lib/neo4j/shared/type_converters.rb

```diff
@@ -102,7 +108,7 @@ def converted_values
         end
 
         def db_type
-          ActiveAttr::Typecasting::Boolean
+          Neo4j::Shared::Boolean
```

Comment:
I've always thought it was weird that we used these constants.  Ideally, I think, they would be symbols like `:boolean`.  But that's a discussion for later ;)